interop/xds/client/client.go

```diff
@@ -58,12 +59,24 @@ var (
 	mu               sync.Mutex
 	currentRequestID int32
 	watchers         = make(map[statsWatcherKey]*statsWatcher)
+
+	// 0 or 1 representing an RPC has succeeded. Use HasRpcSucceeded and
+	// RpcSucceeded to access in a safe manner.
+	rpcSucceeded uint32
 )
 
 type statsService struct {
 	testpb.UnimplementedLoadBalancerStatsServiceServer
 }
 
+func HasRpcSucceeded() bool {
```

Comment:
No need to export this and the other function.

`RPC`, not `Rpc`, otherwise go lint will complain.

And call this `setRPCSucceeded()`?

Reply:
Awesome, I am up to page 3 of a tour of Go now: https://tour.golang.org/basics/3 :-)

Done (I think!)